modules/_labs/select/react/stories/stories_visualTesting.tsx

```diff
@@ -163,159 +157,6 @@ export const SelectStatesOption = () => (
   </div>
 );
 
-const disabledOptions = [
-  {label: 'Carrier Pigeon', value: 'pigeon', disabled: true},
-  {label: 'E-mail', value: 'email'},
-  {label: 'Phone', value: 'phone'},
-  {label: 'Fax', value: 'fax', disabled: true},
-  {label: 'Mail', value: 'mail'},
-  {label: 'Mobile Phone', value: 'mobile_phone'},
-  {label: 'Telegram', value: 'telegram', disabled: true},
-];
-
-const Container = ({children, style = {}, ...elemProps}) => {
-  return (
-    <div
-      style={{
-        border: `1px solid ${colors.soap600}`,
-        marginTop: 50,
-        overflow: 'hidden',
-        padding: 20,
-        ...style,
-      }}
-      {...elemProps}
-    >
-      {children}
-    </div>
-  );
-};
-
-const SelectModal = () => {
-  const {targetProps, modalProps, closeModal} = useModal();
-
-  return (
-    <div>
-      <Button variant={Button.Variant.Primary} {...targetProps}>
-        Show Modal
-      </Button>
-      <Modal heading="Modal with Select" {...modalProps}>
-        <p>The menu for this Select should break out of the Modal.</p>
-        <FormField label="Label" inputId="select-modal">
-          {controlComponent(<Select name="city" options={manyOptions} />)}
-        </FormField>
-        <Button style={{marginRight: '16px'}} onClick={closeModal} variant={Button.Variant.Primary}>
-          Submit
-        </Button>
-        <Button onClick={closeModal} variant={Button.Variant.Secondary}>
-          Cancel
-        </Button>
-      </Modal>
-    </div>
-  );
-};
-
-export const AccessibilityTest = () => (
-  <div>
-    <Container>
-      <p>
-        This Select has its <strong>aria-required</strong> prop set to true. When its listbox menu
-        is opened, we expect aria-required to be set to "true" for the listbox.
-      </p>
-      <FormField label="Label (aria-required)" inputId="select-aria-required" required={true}>
-        {controlComponent(
-          <Select name="select-aria-required" options={options} aria-required={true} />
-        )}
-      </FormField>
-      <p>
-        This Select has its <strong>required</strong> prop set to true. Again, when its listbox menu
-        is opened, we expect aria-required to be set to "true" for the listbox.
-      </p>
-      <FormField label="Label (required)" inputId="select-required" required={true}>
-        {controlComponent(<Select name="select-required" options={options} required={true} />)}
-      </FormField>
-    </Container>
-  </div>
-);
-
-export const DisabledOptionsTest = () => (
-  <div>
-    <Container>
-      <p>Disabled options may not be assistively focused using the keyboard.</p>
-      <FormField label="Label (Disabled Options)" inputId="select-disabled-options">
-        {controlComponent(<Select name="select-disabled-options" options={disabledOptions} />)}
-      </FormField>
-    </Container>
-  </div>
-);
-
-export const PortalTest = () => (
```

Comment:
There were a number of stories in `stories_visualTesting.tsx` that don't require snapshots and never should have been there in the first place (stories used for Cypress and manual testing). I've moved them to `stories_testing.tsx`.